tracker-neuralnet/CMakeLists.txt

```diff
@@ -1,8 +1,5 @@
 include(opentrack-opencv)
 set(host-spec "${CMAKE_SYSTEM_NAME} ${CMAKE_SYSTEM_PROCESSOR} ${CMAKE_SIZEOF_VOID_P}")
-if(host-spec MATCHES "^Linux i[3-6]86 4$")
```

Comment:
This shouldn't be necessary.

See https://github.com/opentrack/opentrack/blob/master/cmake/FindONNXRuntime.cmake#L36

Reply:
Yes, removed in latest patch.